### DISCUSSION TITLE
Running quic proxy

### DISCUSSION THREAD

**Discussion by ffarhanaamin** [blockworks-foundation/lite-rpc#377] [Q&A] [answered]
TAG: v0.2.4

I am trying to run quick-forward-proxy, by invoking the following, 

RUST_LOG=debug cargo run --bin solana-lite-rpc-quic-forward-proxy -- --proxy-listen-addr 0.0.0.0:11111

I am getting the following exception,

Updating git repository `https://github.com/rpcpool/yellowstone-grpc.git`
    Finished dev [unoptimized + debuginfo] target(s) in 0.51s
     Running `target/debug/solana-lite-rpc-quic-forward-proxy --proxy-listen-addr '0.0.0.0:11111'`
Error: Cannot find the identity file provided

Caused by:
    No such file or directory (os error 2)

Caused by:
    No such file or directory (os error 2)

I can read in the documentation that identify file is not a mandatory parameter. 

https://github.com/blockworks-foundation/lite-rpc/tree/main/quic-forward-proxy

**Comment by grooviegermanikus** [comment]
confirmed to be broken (tested commit 1e956a2b)

**Comment by grooviegermanikus** [comment, marked as answer]
merged Fix into main branch - thx for contributing

**Comment by ffarhanaamin** [comment]
I was able to run it successfully, thank you very much. 

I have another quick clarification question. Here is my setup, 

lite-rpc: is connected to a public RPC node, and quick-forward-proxy is configured to "127.0.0.1:1111"
quick-foward-proxy: is running on 127.0.0.1111

Whenever I try to send the transactions to my lite-rpc I can see in the logs of quick-forward-proxy that is is not to send over the transactions to TPU (I am not using a staked validator) and I keep getting these errors in the quic-proxy-log

y::quinn_auto_reconnect: Failed connect initially to target 69..**11228
2024-04-02T21:16:37.686454Z WARN send_tx_batch_to_tpu: solana_lite_rpc_quic_forward_proxy::quinn_auto_reconnect: Failed connect initially to target 141...:8013
2024-04-02T21:16:37.686457Z WARN send_tx_batch_to_tpu: solana_lite_rpc_quic_forward_proxy::quinn_auto_reconnect: Failed connect initially to target 141.98.216.131:8009

(IPs are masked just incase.)

Is it the normal behaviour, are these ports normally filtered for any unwanted traffic? and in order to take advantage of the quic proxy do I need to run my own staked validator?

Rust version: rustc 1.73.0 (cc66ad468 2023-10-03)
